### PR TITLE
Changes investigate logs to use a define

### PR DESCRIPTION
**Pull request by lzimann**
Makes things a bit easier.

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -41,6 +41,7 @@
 #include "code\__DEFINES\language.dm"
 #include "code\__DEFINES\layers.dm"
 #include "code\__DEFINES\lighting.dm"
+#include "code\__DEFINES\logging.dm"
```
Comment (Cyberboss) [inline tgstation.dme]: .

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: tgstation.dme (hunk 1 above)
Comment (lzimann) [inline tgstation.dme]: Oops.

**Review by Cyberboss — APPROVED** [review]

**Comment by Jordie0608** [comment]
Does it?

File: tgstation.dme (hunk 1 above)
Comment (Cyberboss) [inline tgstation.dme]: but... it should

**Comment by lzimann** [comment]
Well, easier to change a single line than multiple when you want to rename an investigate.

**Pull request merged**